freqtrade/exchange/__init__.py

```diff
@@ -234,6 +235,15 @@ def validate_ordertypes(self, order_types: Dict) -> None:
                     'On exchange stoploss is not supported for %s.' % self.name
                 )
 
+    def validate_order_time_in_force(self, order_time_in_force: Dict) -> None:
+        """
+        Checks if order time in force configured in strategy/config are supported
+        """
+        if any(v != 'gtc' for k, v in order_time_in_force.items()):
+            if not self.name == 'Binance':
```

Comment:
in line 233 we check with `if self.name is not 'Binance':` ...
i guess some consistency would not hurt (and it's easier readable than negating a positive check ).